### ISSUE TITLE
Add option to toggle backs from generated interface

### ISSUE THREAD

**Issue by jhauberg** [jhauberg/cards.py#111]
If a project is built with backs, it would be useful being able to toggle them on/off in the front-end interface.

This toggle would have to be hidden in case a project is _not_ built with backs.

**Comment by jhauberg** [comment]
I think this could be implemented simply by adding a special CSS class to the pages containing backs. It probably wouldn't add any styling, but would make it possible to find only those page elements in the DOM.

**Comment by jhauberg** [comment]
One issue with this feature: page numbers would be off (they are generated while building).

Maybe the logo and page number tags could just be removed?

Alternatively, for a visual fix, instead of completely hiding away the pages with backs, they could just be "minimized" (with CSS styles). This still doesn't solve the problem when printing, though.

I suppose the footer could also just be updated using some javascript... Should be simple enough to do a count of all page elements that are not hidden. This does not fix the issue for any cards using the `_page_number` or `_pages_total` field, but honestly, those should be done away with, anyway.

So in conclusion: maybe just remove page numbers from the template, and make dynamically using javascript.
